session-prep-template.md

```diff
@@ -0,0 +1,35 @@
+# [INSERT STORYTELLER NAME] Prep
+
+## Personal information
```

Comment:
This is all captured in the Calendly invitation, so we don't need the facilitator to re-ask the storyteller to provide this information.

Reply:
Yes. I was moving our Google Doc template that we use to prep before a session into a template here. 

Should the change here be an additional annotation in the template as well as a change in wording to the facilitator checklist?

Reply:
I think the updates you made resolve my concern.